app/src/screens/Dialer/__tests__/Dialer.test.tsx

```diff
@@ -1,74 +1,291 @@
 import '@testing-library/jest-native/extend-expect';
 import { render, screen, fireEvent } from '@testing-library/react-native';
 import React from 'react';
+import { Text } from 'react-native';
 import { Provider } from 'react-redux';
-import configureStore from 'redux-mock-store';
+import configureStore, { MockStore } from 'redux-mock-store';
```

Comment:
I thought we were going to not use `redux-mock-store` ?

What if you use `createStore` from https://github.com/twilio/twilio-voice-react-native-app/blob/main/app/src/store/app.ts#L11 ?

Reply:
I spoke with @manjeshbhargav about this and it seems like there were unforeseen issues. I'll let @manjeshbhargav speak to it. It is unfortunate that we could not do it with this PR. 😞

Reply:
@kpchoy I tried integrating the real store, and there were a lot of issues related to the react dom that I don't have the expertise yet to debug. I would love to work on this in a separate initiative.

Reply:
got it!